dspace-spring-rest/src/main/java/org/dspace/app/rest/RestResourceController.java

```diff
@@ -757,14 +757,18 @@ public ResourceSupport findRel(HttpServletRequest request, HttpServletResponse r
 
         } else {
             if (resource.getEmbeddedResources().get(rel) == null) {
-                response.setStatus(HttpServletResponse.SC_NO_CONTENT);
+                if (model.equals("submissiondefinitions")) {
+                    response.setStatus(HttpServletResponse.SC_OK);
+                    return new ResourceSupport();
```

Comment:
this could be the hard part... we need to return an empty collection resource with a proper self link and pagination details (0 results). The name of the embedded empty list is known as it must match the requested "rel" this could eventually simplify the job as we don't need to figure out the type of object that we return